Documentation for default Elasticsearch VDB (#472)
NVIDIA-AI-Blueprints/rag: docs/deploy-docker-self-hosted.md

   docker compose -f deploy/compose/docker-compose-*-server.yaml up -d --build
   ```

> - By default, **Elasticsearch** is deployed as the vector database (`vectordb.yaml` with the default profile). **Milvus is optional**: start it with `docker compose -f deploy/compose/vectordb.yaml --profile milvus up -d` and set `APP_VECTORSTORE_NAME` / `APP_VECTORSTORE_URL` for Milvus (see [Vector database configuration](change-vectordb.md)). When you use the Milvus profile, Milvus can run GPU-accelerated; choose the GPU ID with the variable below. For all service port mappings and GPU assignments, see [Service Port and GPU Reference](service-port-gpu-reference.md).

Reviewer: ```suggestion
By default, Elasticsearch is deployed as the vector database (`vectordb.yaml` with the default profile). Milvus is optional. You can start Milvus with `docker compose -f deploy/compose/vectordb.yaml --profile milvus up -d` and set `APP_VECTORSTORE_NAME` / `APP_VECTORSTORE_URL` for Milvus (see [Vector database configuration](change-vectordb.md)). When you use the Milvus profile, Milvus can run as GPU-accelerated. Choose the GPU ID using the following variables. For all service port mappings and GPU assignments, refer to [Service Port and GPU Reference](service-port-gpu-reference.md).
```
